src/db.ts

```diff
@@ -1,7 +1,26 @@
 import low from "lowdb";
 import FileAsync from "lowdb/adapters/FileAsync";
 
-const adapter = new FileAsync("exchanges.test.json");
+type Exchange = {
+    sender: string,
+    recipient: string,
+    senderMemo: string,
```

Comment:
Since we're going to process exchanges, I think it would be better if we changed this to `xmrAddress`

Reply:
Done ;)